resources/js/features/root/layout/Header.tsx

```diff
@@ -110,9 +110,10 @@ const HeaderRight = () => {
 
 const useCurrentTime = () => {
   const [currentTime, setCurrentTime] = createSignal(new Date());
+  let interval: ReturnType<typeof setInterval>;
   onMount(() => {
-    const interval = setInterval(() => setCurrentTime(new Date()), 1000);
-    return () => clearInterval(interval);
+    interval = setInterval(() => setCurrentTime(new Date()), 1000);
   });
+  onCleanup(() => clearInterval(interval));
   return currentTime;
```

Comment:
faktycznie, ma to sens. Z przyzwyczajenia (react) zrobiłem to w ten sposób. To jest poprawne

Reply:
Tak zgadywałem że react tak ma :)